### PR TITLE
Add 5 chars random suffix to service user names

### PR DESCRIPTION
**Pull request by lepiolet**
## Checklist
- [x] Bump `.version` and `.appVersion` in [k8s/crd/Chart.yaml](../k8s/crd/Chart.yaml)
- [x] Bump `.version` and `.appVersion` in [k8s/chart/Chart.yaml](../k8s/chart/Chart.yaml)

### CODE REVIEW

File: sdp-common/src/service.rs [hunk 1]
```diff
@@ -534,20 +534,33 @@ pub fn needs_injection<A: Annotated>(entity: &A) -> bool {
...
+
 pub fn get_service_username(cluster_name: &str, service_ns: &str, service_name: &str) -> String {
-    format!("{}_{}_{}", cluster_name, service_ns, service_name)
+    format!(
+        "{}_{}_{}_{}",
+        cluster_name,
+        service_ns,
+        service_name,
+        get_random_suffix(5)
+    )
```
Comment (thomascellerier) [inline sdp-common/src/service.rs:553]: We need to check whats the limit on the total size for the name here.

**Review by thomascellerier — APPROVED** [review]
Is there any impact in terms of migrating from an older version?

File: sdp-common/src/service.rs (hunk 1 above)
Comment (mandopaloooza) [reply sdp-common/src/service.rs:553]: Will this cause issue in existing injectors when we try to activate a service user? 

See below
https://github.com/appgate/sdp-k8s-injector/blob/72a81e3fd85c5db47372f1af2c925317fd03e7f8/sdp-identity-service/src/identity_creator.rs#L452-L465

Comment (lepiolet) [reply sdp-common/src/service.rs:553]: 128 chars

**Comment by lepiolet** [comment]
> Is there any impact in terms of migrating from an older version?

None that i can think of

**Pull request merged**